aiven/client/cliarg.py

```diff
@@ -136,7 +136,7 @@ def wrapped(self: CommandLineTool) -> T:
 )
 arg.organization_id = arg("--organization-id", required=True, help="Organization identifier")
 arg.organization_id_positional = arg("organization_id", help="Organization identifier")
-arg.parent_id = arg("--parent-id", help="Organization or account identifier")
+arg.parent_id_mandatory = arg("--parent-id", required=True, help="Organization or account identifier")
```

Comment:
We can keep `parent_id` should here for `project_update` which we should migrate to use `parent-id` as well

Reply:
Addressed